README.md

```diff
@@ -75,6 +75,39 @@ You can specify different template extensions by setting the `template_extension
 template_extensions: ['txt', 'lqd']
 ```
 
+#### Custom paths
```

Comment:
The only reason I think we may want to also include `permalinks` is for compatibility with jekyll. But I do like the name custom paths better. I think we should include the word permalinks in the docs.